src/Fraction.jl

```diff
@@ -823,6 +823,7 @@ that it will always be returned by a call to the constructor when the same
 base ring $R$ is supplied.
 """
 function fraction_field(R::Ring; cached::Bool=true)
+   @req !is_trivial(R) "The zero is currently not supported as a coefficient ring."
```

Comment:
I find "The zero" confusing here. How about
```suggestion
   @req !is_trivial(R) "Zero rings are currently not supported as coefficient ring"
```